src/utils/index.ts

```diff
@@ -333,6 +333,24 @@ export const preparePendingCuepoints = (
   );
 };
 
+function filterPreviewDuplications(sortedData: Array<ItemData>) {
+  const filteredArr = [sortedData[0]];
+  for (let i = 0; i < sortedData.length - 1; i++) {
+    if (
+      !(
+        sortedData[i].title === sortedData[i + 1].title &&
+        sortedData[i].partnerData === sortedData[i + 1].partnerData &&
+        [sortedData[i].tags, sortedData[i + 1].tags].includes(
+          'select-a-thumb, __PREVIEW_CUEPOINT_TAG__'
```

Comment:
if cue-points have specific tag, why we can't filter cue-points only by `'select-a-thumb, __PREVIEW_CUEPOINT_TAG__'` tags without comparing `title` and `partnerData`?